sdk/purview/Microsoft.Azure.Management.Purview/src/Microsoft.Azure.Management.Purview.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.Purview</PackageId>
+    <Description>Azure Purview is a unified data governance service that helps you manage and govern your on-premises, multicloud, and software-as-a-service (SaaS) data. </Description>
+    <Version>1.0.1</Version>
```

Comment:
For first release preview libraries, please start with version number `1.0.0-beta.1`